app/models/history.rb

```diff
@@ -261,21 +261,21 @@ def self.calculated_symptom_onset(patient: nil, created_by: 'Sara Alert System',
                 because a report meeting the symptomatic logic was #{action}."
               end
 
-    create_history(patient, created_by, HISTORY_TYPES[:monitoring_change], comment)
+    create_history(patient, created_by, HISTORY_TYPES[:monitoring_change], comment, create: create)
   end
 
-  def self.send_close_conact_method_blank(patient: nil, created_by: 'Sara Alert System', type: 'Unknown')
+  def self.send_close_conact_method_blank(patient: nil, created_by: 'Sara Alert System', type: 'Unknown', create: true)
```

Comment:
there is a typo in this method `conact` => `contact`

Reply:
I'll fix this in a PR after this one is merged since the issue wasn't introduced in this PR